### DISCUSSION TITLE
Prompt the code by bot status (python).

### DISCUSSION THREAD

**Discussion by FOSS-5head** [Rapptz/discord.py#7226] [Q&A] [answered]

**Comment by scrazzz** [comment, marked as answer]
https://discordpy.readthedocs.io/en/stable/faq.html#how-do-i-set-the-playing-status

If you're new to the library, it's good to refer the documentation in the README.
If you're new to Python, discord.py isn't a beginner friendly library to start off with, it's best to start on something a bit easier.

**Reply by FOSS-5head** [reply]
Hi, thanks for the reply! I'm not new to python, the module was redone, but I didn't know (I already made bots before), thanks.
